prime-angular-frontend/src/app/shared/components/site/party-review/party-review.component.html

```diff
@@ -25,6 +25,11 @@
   {{ party?.email | default }}
 </app-enrollee-property>
 
+<app-enrollee-property *ngIf="$any(party).vendorsSupported !== undefined"
+                       title="Vendor">
+  {{ $any(party).vendorsSupported | healthAuthorityVendor | default }}
```

Comment:
is there any work-around for the `$any()` type casting here? It looks like the input `[party]` is type `Party` but an arbitrary `HealthAuthorityTechnicalSupport` is passed and force casted to check `vendorsSupported`. feels like the input `party` is *polluted* here.  could the the force-casting be eliminated by pass in a separate input param like `healthAutorityTechnicalSupport`?

Reply:
Good point ... I'm trying an alternative to `$any()` but so far not working 

![image](https://user-images.githubusercontent.com/7586665/159546004-31c0b6b9-725a-4bc1-91f8-026dce4043ad.png)

Did you have something in mind @johnnyl6 ?

Reply:
I was thinking of a separate input param to hold the `vendorSupported` property in `PartyReviewComponent`
```typescript
@Input() public healthAutorityTechnicalSupport: HealthAuthorityTechnicalSupport;
```
```html
<app-enrollee-property *ngIf="healthAutorityTechnicalSupport?.vendorsSupported"
                       title="Vendor">
  {{ healthAutorityTechnicalSupport.vendorsSupported | healthAuthorityVendor | default }}
</app-enrollee-property>
```

Reply:
Sorry to jump in here, but this is another place where projection makes it easier. and doesn't cause us to have to add outside knowledge to a generic component. 

![image](https://user-images.githubusercontent.com/59669288/159768240-cd9dad9c-7bb9-4b13-a170-36520798a8c9.png)

So by doing something as simple as a content select on app-enrollee-properties inside the party-review.component.ts. Now whenever we use the component we can wrap additional properties and they are displayed before the address. So then in the health-auth-org-info-page.component, which has more context and knows of the HealthAuthTechSupport you can just do something like this

![image](https://user-images.githubusercontent.com/59669288/159768546-281f85c5-a5c3-4cab-8f91-13ad2cbfae41.png)

As you can see i pass in multiple to show it doesn't limit you to one. Now the shared generic component doesn't have to have the context from it's parent to be able to render this new field.

Reply:
@hogmann Great suggestion!